geoapps/octree_creation/driver.py

```diff
@@ -43,7 +44,6 @@ def run(self) -> Octree:
     def octree_from_params(params: OctreeParams):
         print("Setting the mesh extent")
         entity = params.objects
-
         p_d = [
```

Comment:
this should probably be contained in a method on the OctreeParams class like get_padding

Reply:
..params could also be used to provide the list of cell sizes directly

Reply:
We simplified it from the original implementation to avoid too many parameters. But agreed, I can make a method on params.